apis/fluent-en_GB/atrium-api-fluent-en_GB/src/jvmTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/DateSubjectChangerSamples.kt

```diff
@@ -0,0 +1,51 @@
+package ch.tutteli.atrium.api.fluent.en_GB.samples
+
+import ch.tutteli.atrium.api.fluent.en_GB.*
+import ch.tutteli.atrium.api.verbs.internal.expect
+import java.text.DateFormat
+import java.text.SimpleDateFormat
+import java.time.LocalDate
+import kotlin.test.Test
+
+class DateSubjectChangerSamples {
+
+    private val formatter: DateFormat = SimpleDateFormat("yyyy-MM-dd")
+
+    @Test
+    fun asLocalDateFeature() {
+        val date = formatter.parse("1995-07-17")
+
+        expect(date)
+            .asLocalDate() // subject is now of type LocalDate
+            .toEqual(LocalDate.parse("1995-07-17"))
+
+        fails {
+            expect(date)
+                .asLocalDate()                                       // subject is now of type LocalDate
+                .toBeAfter(LocalDate.parse("2025-07-17"))       // fails
+                .toBeBefore(LocalDate.parse("1996-07-17"))      // not evaluated/reported because `cause` already fails
+                                                                     // use `.asLocalDate { ... }` if you want all assertions evaluated
```

Comment:
```suggestion
                .asLocalDate()                                  // subject is now of type LocalDate
                .toBeAfter(LocalDate.parse("2025-07-17"))       // fails
                .toBeBefore(LocalDate.parse("1996-07-17"))      // not evaluated/reported because `toBeAfter` already fails
                                                                // use `.asLocalDate { ... }` if you want all assertions evaluated
```